openlibrary/core/bookshelves.py

```diff
@@ -121,7 +121,8 @@ def get_users_logged_books(cls, username, bookshelf_id, limit=100, page=1):
 
         username (str) - who logged this book
         work_id (int) - the Open Library work ID as an int (e.g. OL123W becomes 123)
-        bookshelf_id (int) - the ID of the bookshelf, see: PRESET_BOOKSHELVES
+        bookshelf_id (int) - the ID of the bookshelf, see: PRESET_BOOKSHELVES.
```

Comment:
Let's make `bookshelf_id` a keyword arg which defaults to `None` and then we don't need to pass `0` in to the function call (the default will be all bookshelves). This makes more sense than having a reserved integer.

Also, when we do our check below, we'll want to specifically check `is None`.

Reply:
`bookshelf_id` is declared to be an int so I would be more favorable to defaulting to 0 instead of None.

Reply:
I think having a clear value separation between a case where all bookshelves should be considered v. a specific shelf considered is better served by a non-int value `None`, as otherwise the programmer is left to spend cognitive cycles wondering if an `id` or 0 is, could, or may eventually be a valid bookshelf ID.

"If no bookshelf ID is provided, include all records" makes more intuitive sense than, "if the bookshelf_id happens to be the special value 0, which happens to be an integer, then don't actually limit by bookshelf_id 0, but instead include all bookshelves"